x-pack/plugins/cloud_security_posture/server/create_indices/create_indices.ts

```diff
@@ -84,20 +86,25 @@ const createLatestFindingsIndex = async (esClient: ElasticsearchClient, logger:
     const { template, composed_of, _meta } =
       findingsIndexTemplateResponse.index_templates[0].index_template;
 
-    if (template?.settings) {
-      template.settings.lifecycle = {
-        name: '',
-      };
-    }
-
     // We always want to keep the index template updated
     await esClient.indices.putIndexTemplate({
       name: LATEST_FINDINGS_INDEX_TEMPLATE_NAME,
       index_patterns: LATEST_FINDINGS_INDEX_PATTERN,
       priority: 500,
+      template: {
+        mappings: template?.mappings,
+        settings: {
+          ...template?.settings,
+          default_pipeline: CSP_LATEST_FINDINGS_INGEST_TIMESTAMP_PIPELINE,
+          lifecycle: {
```

Comment:
note that you are changing the previous behaviour.
Had you fully checked it? i'm not sure our UT are covering this

Reply:
I can remove this part if needed. I just wanted to keep the same functionality as we had in beforehand. 
It is the same as it used to be (we used to populate the entire template)

Reply:
++ 
please revert to the previous ILM
https://github.com/elastic/kibana/pull/141095/files#diff-f86802a29feadf8c5b8336447ba954343c4b404dcab0785c66ee3a44149f5b78L87-L92

Reply:
Fixed